### PR TITLE
[feature] Export Quays for NeTEx France

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: src/documentation/nl_hellogo_fares_to_ntm_fare_specs.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The fare data expected at the input of the connector comes in the form of XML fi
 
 The supported fare structures depend on origin-destination (OD) stop pairs in two ways:
 - the ticket price between an origin and a destination is directly specified (*DirectPriceMatrix*)
-- the price for a fare distance unit is specified, as well as the fare distance between an origin and a destination (*UnitPrice* & *DirectMatrix*). The ticket price between the origin and destination stops is computed by multiplying the fare distance by the fare distance unit. In this case, one and only one frame of type *UnitPrice* is expected: if none or more than two are provided, the file is considered invalid and should be discarded.
+- the price for a fare distance unit is specified, as well as the fare distance between an origin and a destination (*UnitPrice* & *DistanceMatrix*). The ticket price between the origin and destination stops is computed by multiplying the fare distance by the fare distance unit. In this case, one and only one frame of type *UnitPrice* is expected: if none or more than two are provided, the file is considered invalid and should be discarded.
```
Comment (prhod) [inline src/documentation/nl_hellogo_fares_to_ntm_fare_specs.md]: strange to have this here, but thx for the fix :+1:

File: src/netex_france/stops.rs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                .append(Node::Text(code.to_owned()))
+                .build()
+        })
+    }
+
+    fn generate_centroid(&self, stop_point: &'a StopPoint) -> Result<Element> {
+        // FIXME: String 'EPSG:4326' is failing at runtime (string below is equivalent but works)
+        let from = "+proj=longlat +datum=WGS84 +no_defs"; // See https://epsg.io/4326
+        let to = "EPSG:2154";
+        let converter = Proj::new_known_crs(from, to, None).ok_or_else(|| {
```
Comment (datanel) [inline src/netex_france/stops.rs]: Will we have a performance issue if we instantiate proj for each stop point?

Comment (woshilapin) [reply src/netex_france/stops.rs]: Good point, I should be able to create a field of `StopPointExporter` and initialize it at the construction... I'll rework that.

Comment (woshilapin) [reply src/netex_france/stops.rs]: Done in 60398174f0e609ddee1a4ae80833960800f483b5.

**Pull request merged**